packages/visual-dev/src/components/Icon/Icon.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import styled from 'styled-components'
+import * as SVGs from './SVGs'
+
+type IconProps = OptionProps & StyleProps
+
+interface OptionProps {
+	icon: keyof typeof SVGs
+}
+
+interface StyleProps {
+	color?: string
+	size?: "small" | "medium" | "large" | string
+}
+
+export const default_size = {
+	"small": "12px",
```

Comment:
Just want to double check that these sizes are meaningful and not just arbritrary (i.e. they'll fit in well with our text sizes).